modules/ROOT/pages/install-gke.adoc

```diff
@@ -0,0 +1,64 @@
+= Luna Streaming Installation on Google Kubernetes Environment
+
+This document will cover installation of Luna Streaming on the Google Kubernetes Environment, or GKE. 
+
+== Prerequisites 
+
+* Helm version 3
+* A Kubernetes cluster 
+* No less than 16 gigs of RAM and 8 virtual processor cores (4 physical cores)
+
+For more on Helm, see https://helm.sh/docs/[Helm documentation].
+For more on Minikube, see https://minikube.sigs.k8s.io/docs/start/[Minikube documentation]. 
+
+== Provision Your GKE Cluster
+
+Before you can install the Helm chart, you need to configure the storage class settings for your cloud provider. The handling of storage varies from cloud provider to cloud provider.
+
+Create a new file called `storage_values.yaml` for the storage class settings. This allows you to match the optimum storage type to the volume.
+If you have specific storage class requirement, for example fixed IOPS disks in AWS, you can have the chart configure the storage classes for you.
+Here is an example `storage-values.yaml` for GKE.
+
+----
+# For GKE
+# default_storage:
+#   provisioner: kubernetes.io/gce-pd
+#   type: pd-ssd
+#   fsType: ext4
+#   extraParams:
+#      replication-type: none
+----
+
+See this https://github.com/datastax/pulsar-helm-chart/blob/master/helm-chart-sources/pulsar/values.yaml[values file] for more details on the settings in `values.yaml`.
+
+Additional provisioning set-up options are available https://docs.k8ssandra.io/install/gke/[here].
+
+== Install Luna Streaming to GKE
+
+Once you have added your storage settings to the `storage-values.yaml` file, install the Helm chart. 
+
+. Create the namespace; in this example, we use `pulsar`.
```

Comment:
Don't think this is needed

Reply:
@mendonk Please make these changes or resolve if you already have. Thanks!